frame/composable-traits/src/defi.rs

```diff
@@ -205,7 +207,7 @@ pub trait SellEngine<Configuration>: DeFiEngine {
 }
 
 pub trait DeFiComposableConfig: frame_system::Config {
-	type MayBeAssetId: AssetIdLike + MaybeSerializeDeserialize + Default;
+	type MayBeAssetId: AssetIdLike + MaybeSerializeDeserialize + Default + MaxEncodedLen + Debug;
```

Comment:
I think better to use `RuntimeDebug` instead of `Debug`

Reply:
there is not such trait